extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/AccessRequest.scala

```diff
@@ -68,17 +70,43 @@ object AccessRequest {
     req
   }
 
+  private val getRolesFromUserAndGroupsMethod: Option[UnboundMethod] =
+    getMethod(
+      SparkRangerAdminPlugin.getClass,
```

Comment:
```suggestion
      classOf[SparkRangerAdminPlugin],
```

Reply:
`SparkRangerAdminPlugin` is an object and does not seem to work with `classOf`.